build.gradle

```diff
@@ -16,29 +16,25 @@ buildscript {
 
     dependencies {
         classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
-        classpath "net.corda.plugins:publish-utils:$corda_gradle_plugins_version"
         classpath "net.corda.plugins:cordformation:$corda_gradle_plugins_version"
         classpath "net.corda.plugins:quasar-utils:$corda_gradle_plugins_version"
-        classpath "org.jetbrains.kotlin:kotlin-noarg:$kotlin_version"
     }
 }
 
-apply plugin: 'kotlin'
-apply plugin: 'kotlin-jpa'
-apply plugin: 'idea'
-apply plugin: 'net.corda.plugins.publish-utils'
-apply plugin: 'net.corda.plugins.cordformation'
-apply plugin: 'net.corda.plugins.quasar-utils'
-apply plugin: 'maven-publish'
-
 repositories {
     mavenLocal()
-    mavenCentral()
     jcenter()
+    mavenCentral()
     maven { url 'https://dl.bintray.com/kotlin/exposed' }
     maven { url 'https://jitpack.io' }
+    maven { url 'https://ci-artifactory.corda.r3cev.com/artifactory/corda-releases' }
 }
 
+apply plugin: 'java'
```

Comment:
do we need the java plugin?

Reply:
No. I think the Kotlin plugin actually applies the Java plugin anyway.